ynput/ayon-core
# Loader: Show statuses in loader

## Changelog Description
Loader tool shows status name or status short name of version.

## Additional info
Added `status` to `VersionItem` which is a status name. Projects model now have method to retrieve list of `StatusItem` for a project. A short name of status is used if name does not fit the column. An icon is not displayed as that would require to implement MaterialIcon for Qt.

## Screenshot
![image](https://github.com/ynput/ayon-core/assets/43494761/32049848-02f0-4305-8b2c-b594b8f4f504)

## Testing notes:
1. Open loader tools (from tray or in DCC).
2. Select project and folder.
3. Products view should display status next to version.
